storybook/stories/API/component/Label.stories.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { Args } from '@storybook/react';
```

Comment:
Hi @wanisramdani , thanks for the PR! For the next one you can also use types from here: https://github.com/recharts/recharts/blob/master/storybook/StorybookArgs.ts

No need to change this PR though - if it works it's all good